### PR TITLE
Fix Issue #26

### PR DESCRIPTION
**Pull request by samcarlos**
Adding score function for mlp_classifier

### CODE REVIEW

File: muffnn/mlp/mlp_classifier.py [hunk 1]
```diff
@@ -262,3 +262,8 @@ def __getstate__(self):
             state['n_classes_'] = self.n_classes_
 
         return state
+
+    def score(self, X, y):
+        accuracy = np.array(y) == self.predict(X)
+        accuracy = accuracy[np.array(y) != -1].mean(axis=0).mean()
```
Comment (beckermr) [inline muffnn/mlp/mlp_classifier.py]: This average is per label. I do not remember. Isn't the loss computed as an average over the non-missing labels for each example and then an average over the examples? We might want to do that here too if so.

Comment (samcarlos) [inline muffnn/mlp/mlp_classifier.py]: Right now, the loss function takes a mean of errors without regard to variable:

`        if self.multilabel_:
            cross_entropy = tf.nn.sigmoid_cross_entropy_with_logits(
                t, tf.cast(self.input_targets_, np.float32))
`
`        self._obj_func = tf.reduce_mean(cross_entropy)
`
If we were to change it, I think we want to make the columns of response equal in weight and so this score function should do that. What do you think?

Comment (beckermr) [inline muffnn/mlp/mlp_classifier.py]: Ahh yes. I remember now. I think rows, you think columns, and we are doing none of those. OK. Let's merge as is, or go for columns. I will leave the decision up to you, but merging as is would be fine by me. 

Thanks for bearing with my poor memory!

**Comment by beckermr** [comment]
LGTM!

File: muffnn/mlp/mlp_classifier.py (hunk 1 above)
Comment (beckermr) [inline muffnn/mlp/mlp_classifier.py]: WAIT! Don't merge yet.
